bug fix in double and triple sine

SimonSchmitt88 opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

When thinking of a Double or Triple - sine, I would expect the sum of three or two sinuses and not the product.

## Description
<!--- Describe your changes in detail -->

Changed the product to a sum

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

When a signal with two or three different frequencies should be applied the result is the sum of the three individual signals 

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, tests ran to see how -->
<!--- your change affects other areas of the code, etc. -->

Tested on setup 3.

## Screenshots (only if appropriate, delete if not):

## Types of changes
<!--- What types of changes does your code introduce? Put an 'x' in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
<!--- Go over all the following points, and put an 'x' in all the boxes that apply. -->
<!--- If you're unsure about any of these, ask. -->
- [x] My code follows the code style of this project.
- [ ] I have documented my changes in the changelog (`documentation/changelog.md`)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have checked that the change does not contain obvious errors (syntax, indentation, mutable default values).
- [ ] I have tested my changes using 'Load all modules' on the default dummy configuration with my changes included.
- [ ] All changed Jupyter notebooks have been stripped of their output cells.

Neverhorst commented:
It was a sum of sines before (like a year ago or so) and then people (I think it was @kay-jahnke, sorry if not) urged me to change it to a product.
Please settle on one solution and define a use case for that before we change it back in another year...

SimonSchmitt88 commented:
Thanks for your comments. For me it seems most user-friendly, that a signal, generated by entering three different frequencies, also consists of these frequencies (by means of FT).

An example, when a linear superposition is needed, is if two hyperfine transitions (freq1 and freq2) should be driven simultanously. 

Anyway, I added the product version again and changed the function names accordingly. In the other part of the main branch, I only found one appearence of these functions ('_get_multiple_mw_element' in 'pulse_objects.py') and here I changed to the sum version (for the reasons described above).

I hope this is fine for everyone. If not please let me know.
